Android building and installation instructions (#2172)
diasurgical/devilutionX: docs/installing.md

</details>

<details><summary>Android</summary>

- Copy the APK file to device and tap on it on device's file explorer or install via `adb install` (if USB debugging is enabled on device).

> - Copy the MPQ files to `devilutionx` folder in root of phone's internal memory or SD card. Create the folder if it doesn't exist.

Reviewer: ```suggestion
- Create a folder called `devilutionx` in the root of the phone's internal memory or SD card, and place the MPQ-files there.
```

Author: The thought was that APK will create the folder upon first launch
But your suggestion works until this is implemented :)
